README.md

```diff
@@ -1 +1,25 @@
-# json-schema-form
\ No newline at end of file
+<p align="center">
+<img src=".github/media/jsf_logo_dark.png"  width="600" alt="json-schema-form">
+</p>
+
+<p align="center">
+  <code>json-schema-form</code> is a headless UI form library powered by <a href="https://json-schema.org/">JSON Schemas</a>. 
+<br/>
+  It transforms JSON schemas into Javascript to be consumed by your UI libraries.
+</p>
+
+---
+
+### Why JSON Schemas for forms?
+
+JSON Schemas are the SSoT (Single Source of Truth) that allows you to share form's _structure_ and _validations_ between frontend and backend, regardless of the language used.
+
+## Documentation
+
+Check the 📚 **[JSF website](https://json-schema-form.vercel.app/)** for documentation and demos.
```

Comment:
Might also be beneficial to add a link here to the JSON schema docs / website.

Reply:
The link is above at line 6 :D

Reply:
lol oops 😬